Backend/BikesRentalServer.DataAccess/Repositories/MalfunctionsRepository.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BikesRentalServer.DataAccess.Repositories.Abstract;
+using BikesRentalServer.Models;
+using Microsoft.EntityFrameworkCore;
+
+namespace BikesRentalServer.DataAccess.Repositories
+{
+    public class MalfunctionsRepository : IMalfunctionsRepository
+    {
+        private readonly DatabaseContext _dbContext;
+
+        public MalfunctionsRepository(DatabaseContext dbContext)
+        {
+            _dbContext = dbContext;
+        }
+        
+        public IEnumerable<Malfunction> GetAll()
+        {
+            throw new System.NotImplementedException();
```

Comment:
Is it not to be implemented (e.g. not needed)? Maybe use `NotSupportedException` instead?
Also may add `using System`